.github/workflows/docker-deploy.yml

```diff
@@ -0,0 +1,26 @@
+name: Docker Deploy
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: SSH to server
+        uses: appleboy/ssh-action@v1
+        with:
+          host: ${{ secrets.VPS_HOST }}
+          username: ${{ secrets.VPS_USER }}
+          password: ${{ secrets.VPS_PASS }}
```

Comment:
Suggestion, advocate for best practices by using SSH keys instead of username & password
The key can also be stored in the secrets 
Example from the action docs https://github.com/appleboy/ssh-action?tab=readme-ov-file#using-private-key-authentication

Reply:
I agree. I will do that isA.